packages/shared/src/utils/constants/contract.ts

```diff
@@ -326,7 +326,7 @@ export const getCardLabelsV2 = () =>
         CURRENT_STAKE: localize('Current stake'),
         DAY: localize('day'),
         DAYS: localize('days'),
-        DEAL_CANCEL_FEE: localize('Deal cancel. fee'),
+        DEAL_CANCEL_FEE: localize('Deal cancellation fees'),
```

Comment:
Is this change intended? Besides cancellation, it also changes `fee` to `fees`

Reply:
The figma has "Deal cancellation fees" as the label now